Jake.ts

```diff
@@ -35,7 +35,16 @@ export function Exec(cmd: string | string[], callback, isSilent?: boolean) {
   // isSilent || console.log(cmd);
   Log(cmdArray, 0);
   if (isSilent && cmdArray.length === 1) {
-    ChildProcess.exec(cmdArray[0], callback);
+    let command = cmdArray[0].split(' ');

```

Comment:
The error with "missing .ts" file is due to the white spaces in the cmd and therefore confusing arguments. You can do cmdArray[0].trim().replace(/\s\s+/g,' ').split(' ') instead.